src/element-action.html

```diff
@@ -53,13 +58,21 @@
             <option value="click">Click</option>
             <option value="clear">Clear</option>
             <option value="sendKeys">Send Keys</option>
+            <option value="getValue">Get Value</option>
+            <option value="getText">Get Text</option>
+            <option value="getAttribute">Get Attribute</option>
+            <option value="takeScreenShot">Element Screenshot</option>
+            <option value="hover">Hover</option>
         </select>
     </div>
     <div class="form-row" id="actionSendKeys">
         <label for="node-input-sendKeys"><i class="fa fa-tasks"></i> Text to Send</label>
         <input id="node-input-sendKeys" type="text">
     </div>
-
+    <div class="form-row" id="get_Attribute">
```

Comment:
For consistency, this should be named `getAttribute` instead of `get_Attribute` (and update your scripts above to reflect this change)

Reply:
changed as suggested